### ISSUE TITLE
✨ Display 'Datepicker' above input field prop instead of always below 

### ISSUE THREAD

**Issue by cmjoseph07** [svelte-plugins/datepicker#22]
**Is your feature request related to a problem? Please describe.**
The is a bit of frustration when pulling a datepicker at the bottom of a clients view. The datepicker always spawns under the input element even if spawning said element goes out of view. 

**Describe the solution you'd like**
As 'align' I feel a prop for top true or not would allow the datepicker to spawn above said element if the user required it and could even change it programmically.

**Describe alternatives you've considered**
All I have been able to do is move elements higher on the screen but this can cause where workflows were date fields are not expected until later.

**Comment by guacamole** [comment]
@cmjoseph07  I'm looking for a fix in this area as well. I love the implementation it is better than most out there with just this issue with position. Have you found an alternative to this?

**Comment by cmjoseph07** [comment]
@guacamole No, I haven't found a good alt for this yet. Additionally this datepicker seems to be hard to position in general honestly.

**Comment by dysfunc** [comment]
There's a new CSS var `--datepicker-container-top` you can override to control horizontal placement

**Comment by dysfunc** [comment]
I'll work on auto-positioning the calendar based on viewport
